docs/resources/instance_server.md

```diff
@@ -176,6 +197,17 @@ attached to the server. Updates to this field will trigger a stop/start of the s
 - `project_id` - (Defaults to [provider](../index.md#project_id) `project_id`) The ID of the project the server is associated with.
 
 
+## Private Network
+
+~> **Important:** Updates to `private_network` will recreate the attachment Instance.
+
+- `pn_id` - (Required) The private network ID where to connect.
+- `mac_address` The private NIC MAC address.
+- `status` The private NIC state.
+- `zone` - (Defaults to [provider](../index.md#zone) `zone`) The [zone](../guides/regions_and_zones.md#zones) in which the server should be created.
```

Comment:
Maybe we should precise that private networks cannot be used across zones. You can only attach an instance in the same zone as a private network.